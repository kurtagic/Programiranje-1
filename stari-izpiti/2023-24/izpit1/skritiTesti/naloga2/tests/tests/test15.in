1 6 43 400
1 29 84
3 29 78
3 29 6
5 29 176
0 12 10
0 12 72
3 29 139
0 12 81
1 12 4
0 12 138
3 1 180
1 21 96
3 1 58
3 10 64
3 21 152
0 29 234
0 10 173
1 12 52
3 1 199
1 10 52
3 10 115
5 10 11
3 22 150
4 10 178
4 22 229
3 22 119
0 10 52
4 33 217
2 1 184
3 21 35
1 1 42
1 22 209
4 10 14
2 1 106
4 29 202
1 40 122
3 21 95
3 0 174
0 29 197
4 19 219
2 26 10
4 26 168
2 5 141
5 0 212
4 0 112
4 1 85
3 29 150
2 21 160
4 26 84
2 1 29
1 31 162
3 10 67
1 28 187
0 10 196
2 40 41
4 40 207
0 22 112
3 1 172
4 17 221
4 12 152
2 17 72
3 30 164
3 10 105
0 28 206
0 21 111
5 19 125
2 19 210
4 34 87
1 22 177
3 10 72
0 12 232
3 1 88
0 40 155
1 29 207
0 33 81
5 1 49
4 12 188
3 10 219
5 0 146
4 29 16
4 1 11
2 34 185
3 1 162
5 19 3
0 9 103
2 42 175
2 12 145
3 10 157
2 3 189
2 31 112
3 22 168
1 23 113
4 10 207
1 8 53
1 3 222
3 24 39
5 3 103
4 33 10
4 31 77
1 10 69
1 5 110
1 10 148
0 5 50
5 38 185
1 1 172
3 13 209
2 17 220
1 13 171
3 41 166
2 1 4
5 34 234
3 1 204
0 12 51
1 8 106
4 38 96
3 15 159
4 23 106
2 12 10
3 0 4
1 38 149
1 40 222
2 5 44
4 9 77
0 12 40
3 21 74
4 0 220
5 23 236
1 22 75
4 0 184
2 26 51
4 17 182
2 24 152
2 24 6
0 1 4
3 29 140
1 30 65
4 1 81
1 10 148
0 0 232
3 23 48
4 10 211
3 33 191
1 9 57
2 42 199
4 5 21
2 21 146
3 0 196
3 3 59
4 40 201
0 33 180
5 1 232
1 24 146
5 17 86
1 17 142
3 23 144
5 24 230
2 10 181
3 33 215
4 28 173
0 40 72
5 31 28
1 30 75
3 22 95
0 28 199
2 17 18
5 37 199
5 1 196
2 26 154
4 32 90
0 34 48
5 5 185
1 30 133
2 5 98
2 24 101
3 29 19
2 28 179
0 15 86
1 24 150
2 42 157
0 26 212
1 0 36
5 1 21
0 31 107
3 9 197
3 5 228
1 0 47
3 17 162
1 30 33
1 26 198
3 5 89
4 5 122
1 38 180
4 34 232
2 35 50
2 1 110
3 10 226
2 38 214
1 0 191
3 1 60
2 24 38
4 35 40
1 21 233
1 1 29
1 19 221
3 28 108
4 24 67
2 5 138
3 28 31
2 17 77
5 35 130
3 42 106
2 25 57
2 35 134
0 17 206
2 9 217
3 21 156
4 36 173
0 4 145
3 35 191
3 10 71
1 17 48
0 25 199
5 13 176
5 5 10
0 4 29
1 8 95
5 28 30
1 4 20
1 3 159
3 42 6
5 1 81
5 13 164
0 10 142
5 15 22
2 17 217
4 4 56
2 24 188
5 34 94
1 26 224
5 17 127
0 15 168
0 26 73
3 4 215
3 3 114
0 35 61
3 4 140
1 15 204
2 28 173
1 26 35
5 38 219
0 13 153
1 3 184
5 28 34
5 9 21
0 35 71
4 9 176
3 17 146
2 12 195
0 42 97
1 28 92
0 36 153
4 10 71
2 14 17
3 35 43
2 3 131
5 1 69
2 12 215
1 17 137
4 3 115
5 12 141
1 33 45
1 29 157
2 31 10
1 30 196
4 21 96
2 10 47
3 34 62
0 1 229
2 18 150
0 17 118
3 0 124
3 19 141
3 8 140
3 19 84
4 12 121
0 1 18
1 25 72
5 10 93
1 2 171
1 28 147
3 37 208
0 9 86
4 40 166
3 3 34
4 29 134
3 4 142
4 13 9
5 4 64
3 17 49
3 14 69
2 13 82
1 17 177
0 5 205
3 36 148
0 36 191
1 12 72
4 17 188
3 2 238
1 9 79
0 17 127
1 21 19
2 17 45
1 34 133
1 3 209
5 13 156
1 13 58
5 42 141
3 30 92
3 0 11
5 13 93
1 1 165
2 4 141
1 26 189
3 30 208
1 24 90
2 4 103
1 37 203
1 35 216
1 2 145
0 10 198
2 26 237
0 26 155
2 35 169
3 29 92
0 21 38
5 4 110
2 3 6
3 9 76
0 40 150
1 36 69
5 35 137
2 42 231
4 5 72
5 18 81
2 4 71
2 13 32
4 35 123
2 15 26
1 14 91
1 2 216
0 13 87
1 25 84
1 16 156
2 5 82
0 36 100
2 28 218
3 32 108
4 10 13
2 30 81
4 3 136
2 16 91
4 5 206
4 42 53
2 9 131
0 10 206
5 37 223
0 2 124
1 17 15
5 42 189
2 15 200
4 24 8
1 16 124
2 27 55
1 4 5
5 0 158
0 37 176
2 2 225
4 22 215
4 21 89
3 9 110
1 42 192
1 26 232
2 10 50
5 34 19
2 8 49
5 31 123
3 0 188
0 30 97
0 12 75
1 37 235
0 40 207
1 36 87
1 31 213
1 26 78
5 2 208
5 3 117
3 35 71
5 28 165
2 25 115
0 31 158
