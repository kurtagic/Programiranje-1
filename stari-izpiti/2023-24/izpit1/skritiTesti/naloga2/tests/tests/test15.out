[232, 232, 238, 222, 215, 228, 0, 0, 140, 217, 226, 0, 232, 209, 91, 204, 156, 221, 150, 221, 0, 233, 229, 236, 230, 199, 237, 55, 218, 234, 208, 213, 108, 217, 234, 216, 191, 235, 219, 0, 222, 166, 231]
